<p>Fernandes, AD, Macklaim, JM, Linn, TG, Reid, G, Gloor, GB (2013).
&ldquo;*** ANOVA-Like Differential Gene Expression Analysis of Single-Organism and Meta-RNA-Seq.&rdquo;
<em>*** PLoS ONE, 2013, volume 8, issue 7, e67019</em>.
<a href="***%20http://dx.doi.org/10.1371/journal.pone.0067019">*** http://dx.doi.org/10.1371/journal.pone.0067019</a>. 
</p>

<p>Fernandes, D. A, Reid, J., Macklaim, M. J, McMurrough, T.A, Edgell, D.R., Gloor, B. G (2014).
&ldquo;*** Unifying the analysis of high-throughput sequencing datasets: characterizing RNA-seq, 16S rRNA gene sequencing and selective growth experiments by compositional data analysis.&rdquo;
<em>*** Microbiome, 2014, volume 2, 15</em>.
<a href="***%20http://dx.doi.org/10.1186/2049-2618-2-15">*** http://dx.doi.org/10.1186/2049-2618-2-15</a>. 
</p>

<p>Gloor GB, Macklaim JM, Fernandes AD (2016).
&ldquo;*** Displaying Variation in Large Datasets: a Visual Summary of Effect Sizes.&rdquo;
<em>*** Journal of Computational and Graphical Statistics,</em>.
<a href="***%20http://dx.doi.org/10.1080/10618600.2015.1131161">*** http://dx.doi.org/10.1080/10618600.2015.1131161</a>. 
</p>

<p>Nixon MP, Letourneau J, David LA, Mukherjee S, Silverman JD (2022).
&ldquo;*** A Statistical Analysis of Compositional Surveys.&rdquo;
<em>*** ArVix Preprint: 2201.03616,</em>.
<a href="***%20https://arxiv.org/abs/2201.03616">*** https://arxiv.org/abs/2201.03616</a>. 
</p>
