<p>Xu Z, Niu L, Li L, Taylor J (2016).
&ldquo;ENmix: a novel background correction method for Illumina HumanMethylation450 BeadChip.&rdquo;
<em>Nucleic Acids Research</em>. 
</p>

<p>Xu Z, Niu L, Sandler D, Taylor J (2022).
&ldquo;The ENmix DNA methylation analysis pipeline for Illumina BeadChip and comparisons to 7 other preprocessing pipelines.&rdquo;
<em>Clinical Epigenetics</em>. 
</p>

<p>Niu L, Xu Z, Taylor J (2016).
&ldquo;RCP: a novel probe design bias correction method for Illumina Methylation BeadChip.&rdquo;
<em>Bioinformatics</em>. 
</p>

<p>Xu Z, Taylor J, Leung Y, Ho S, Niu L (2016).
&ldquo;oxBS-MLE: An efficient method to estimate 5-methylcytosine and 5-hydroxymethylcytosine in paired bisulfite and oxidative bisulfite treated DNA.&rdquo;
<em>Bioinformatics</em>. 
</p>

<p>Xu Z, Taylor J, Langie S, Boever P, Niu L (2017).
&ldquo;RELIC: a novel dye-bias correction method for Illumina Methylation BeadChip.&rdquo;
<em>BMC Genomics</em>. 
</p>

<p>Xu Z, Taylor J (2020).
&ldquo;Reliability of DNA methylation measures using Illumina methylation BeadChip.&rdquo;
<em>Epigenetics</em>. 
</p>

<p>Xu Z, Xie C, Taylor J, Niu L (2020).
&ldquo;ipDMR: Identification of differentially methyl-ated regions with interval P values.&rdquo;
<em>Bioinformatics</em>. 
</p>

<p>Xu Z, Niu L, Kresovich J, Taylor J (2024).
&ldquo;Methscore: a comprehensive R function for DNA methylation-based health predictors.&rdquo;
<em>Bioinformatics</em>. 
</p>
