<p>Zheng SC, Breeze CE, Beck S, Teschendorff AE (2018).
&ldquo;Identification of differentially methylated cell-types in Epigenome-Wide Association Studies.&rdquo;
<em>Nature Methods</em>, <b>15</b>(12), 1059. 
</p>

<p>Zheng SC, Webster AP, Dong D, Feber A, Graham DG, Sullivan R, Jevons S, Lovat LB, Beck S, Widschwendter M, Teschendorff AE (2018).
&ldquo;A novel cell-type deconvolution algorithm reveals substantial contamination by immune cells in saliva, buccal and cervix.&rdquo;
<em>Epigenomics</em>, <b>10</b>(7). 
</p>

<p>Teschendorff AE, Breeze CE, Zheng SC, Beck S (2017).
&ldquo;A comparison of reference-based algorithms for correcting cell-type heterogeneity in Epigenome-Wide Association Studies.&rdquo;
<em>BMC bioinformatics</em>, <b>18</b>(1), 105. 
</p>

<p>Houseman EA, Accomando WP, Koestler DC, Christensen BC, Marsit CJ, Nelson HH, Wiencke JK, Kelsey KT (2012).
&ldquo;DNA methylation arrays as surrogate measures of cell mixture distribution.&rdquo;
<em>BMC bioinformatics</em>, <b>13</b>(1), 86. 
</p>

<p>Newman AM, Liu CL, Green MR, Gentles AJ, Feng W, Xu Y, Hoang CD, Diehn M, Alizadeh AA (2015).
&ldquo;Robust enumeration of cell subsets from tissue expression profiles.&rdquo;
<em>Nature Methods</em>, <b>12</b>(5), 453. 
</p>

<p>Teschendorff AE, Zheng SC (2017).
&ldquo;Cell-type deconvolution in epigenome-wide association studies: a review and recommendations.&rdquo;
<em>Epigenomics</em>, <b>9</b>(5), 757. 
</p>
