<p>Marini F, Linke J, Binder H (2020).
&ldquo;ideal: an R/Bioconductor package for Interactive Differential Expression Analysis.&rdquo;
<em>BMC Bioinformatics</em>, <b>21</b>, 565.
<a href="https://doi.org/10.1186/s12859-020-03819-5">doi:10.1186/s12859-020-03819-5</a>, <a href="https://bioconductor.org/packages/ideal/">https://bioconductor.org/packages/ideal/</a>. 
</p>

<p>Ludt A, Ustjanzew A, Binder H, Strauch K, Marini F (2022).
&ldquo;Interactive and Reproducible Workflows for Exploring and Modeling RNA-seq Data with pcaExplorer, ideal, and GeneTonic.&rdquo;
<em>Current Protocols</em>, <b>2</b>(4), e411.
<a href="https://doi.org/10.1002/cpz1.411">doi:10.1002/cpz1.411</a>. 
</p>
