<p>Lee S, Loo C, Prasasya R, Bartolomei M, Kohli R, Zhou W (2024).
&ldquo;Low-input and single-cell methods for Infinium DNA methylation BeadChips.&rdquo;
<em>Nucleic acids research</em>.
ISSN 1362-4962, <a href="https://doi.org/10.1093/nar/gkae127">doi:10.1093/nar/gkae127</a>. 
</p>

<p>Kaur D, Lee S, Goldberg D, Spix N, Hinoue T, Li H, Dwaraka V, Smith R, Shen H, Liang G, Renke N, Laird P, Zhou W (2023).
&ldquo;Comprehensive evaluation of the Infinium human MethylationEPIC v2 BeadChip.&rdquo;
<em>Epigenetics Communications</em>, <b>3</b>(1), 6. 
</p>

<p>Ding W, Kaur D, Horvath S, Zhou W (2023).
&ldquo;Comparative epigenome analysis using Infinium DNA methylation BeadChips.&rdquo;
<em>Briefings in bioinformatics</em>, <b>24</b>(1).
ISSN 1477-4054, <a href="https://doi.org/10.1093/bib/bbac617">doi:10.1093/bib/bbac617</a>, 36617464. 
</p>

<p>Zhou W, Hinoue T, Barnes B, Mitchell O, Iqbal W, Lee SM, Foy KK, Lee K, Moyer EJ, VanderArk A, Koeman JM, Ding W, Kalkat M, Spix NJ, Eagleson B, Pospisilik JA, SzabÃ³ PE, Bartolomei MS, Schaaf NAV, Kang L, Wiseman AK, Jones PA, Krawczyk CM, Adams M, Porecha R, Chen BH, Shen H, Laird PW (2022).
&ldquo;DNA methylation dynamics and dysregulation delineated by high-throughput profiling in the mouse.&rdquo;
<em>Cell genomics</em>, <b>2</b>(7).
ISSN 2666-979X, <a href="https://doi.org/10.1016/j.xgen.2022.100144">doi:10.1016/j.xgen.2022.100144</a>, 35873672. 
</p>

<p>Zhou W, Triche TJ, Laird PW, Shen H (2018).
&ldquo;SeSAMe: reducing artifactual detection of DNA methylation by Infinium BeadChips in genomic deletions.&rdquo;
<em>Nucleic Acids Research</em>, <b>gky691</b>.
<a href="https://doi.org/10.1093/nar/gky691">doi:10.1093/nar/gky691</a>. 
</p>
