<p>Gaidatzis D, Lerch A, Hahne F, Stadler MB (2015).
&ldquo;QuasR: Quantification and annotation of short reads in R.&rdquo;
<em>Bioinformatics</em>, <b>31</b>(7), 1130&ndash;1132.
<a href="https://doi.org/10.1093/bioinformatics/btu781">doi:10.1093/bioinformatics/btu781</a>, PMID:25417205. 
</p>

<p>Langmead B, Trapnell C, Pop M, Salzberg SL (2009).
&ldquo;Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.&rdquo;
<em>Genome Biology</em>, <b>10</b>(3), R25.
<a href="https://doi.org/10.1186/gb-2009-10-3-r25">doi:10.1186/gb-2009-10-3-r25</a>, PMID:19261174. 
</p>

<p>Au KF, Jiang H, Lin L, Xing Y, Wong WH (2010).
&ldquo;Detection of splice junctions from paired-end RNA-seq data by SpliceMap.&rdquo;
<em>Nucleic Acids Research</em>, <b>38</b>(14), 4570&ndash;4578.
<a href="https://doi.org/10.1093/nar/gkq211">doi:10.1093/nar/gkq211</a>, PMID:20371516. 
</p>

<p>Kim D, Langmead B, Salzberg SL (2015).
&ldquo;HISAT: a fast spliced aligner with low memory requirements.&rdquo;
<em>Nature Methods</em>, <b>12</b>, 357&ndash;360.
<a href="https://doi.org/10.1038/nmeth.3317">doi:10.1038/nmeth.3317</a>, PMID:25751142. 
</p>
